flutter/lib/src/flutter_enricher_event_processor.dart

```diff
@@ -111,8 +111,7 @@ class FlutterEnricherEventProcessor extends EventProcessor {
   SentryCulture _getCulture(SentryCulture? culture) {
     // The editor says it's fine without a `?` but the compiler complains
     // if it's missing
-    // ignore: invalid_null_aware_operator
-    final languageTag = _window?.locale?.toLanguageTag();
+    final languageTag = _window?.locale.toLanguageTag();
```

Comment:
I believe this is because of an API change in some of the early 2.x Flutter versions. It should be kept as is, otherwise Sentry does not work with Flutter 2.0 anymore.

Reply:
Ok, Flutter 1.17 is currently in the pubspec.yaml - this should probably be run in CI then, might be a couple of things failing.

Reply:
Right now we run on different channels, but not on the minimal version, yes, See https://github.com/getsentry/sentry-dart/issues/636

Reply:
I'd rather keep as it is for compatibility.

Reply:
Yea I'll revert this.

Reply:
Reverted